test/unit/shim/transaction-shim.test.js

```diff
@@ -852,7 +852,7 @@ tap.test('TransactionShim', function (t) {
         shim.insertCATRequestHeaders(headers)
 
         t.equal(tx.pathHashes.length, 1)
-        t.equal(tx.pathHashes[0], '14cd4d06')
+        t.equal(tx.pathHashes[0], '0f9570a6')
```

Comment:
this changed because transaction shim previously didn't default transaction to `''` when undefined whereas the http-outbound did. This slight change resulted in a diff hash for the diff bits of the pathHashes

Reply:
Looking through the spec, it seems like making that divergence should be fine/safe. Also, I'm not sure when we'd have a null transaction name for non-test scenarios so hopefully that would be an extremely rare.

That being said, this particular test has a path appended to the nameState so seems like it should have a name. Oh, its probably because runInTransaction probably creates a non-web by default. The namestate naming only gets applied when web and there's a URL.